content/20.pharmaceuticals.md

```diff
@@ -517,22 +541,26 @@ These findings support the results of small clinical trials that have found fluv
 It has been found to be protective against septic shock in an _in vivo_ mouse model [@doi:10.1126/scitranslmed.aau5266].
-However, it is possible that fluvoxamine also exerts an antiviral effect [@doi:10.3389/fphar.2021.652688].
+It is possible that fluvoxamine also exerts an antiviral effect [@doi:10.3389/fphar.2021.652688].
 Thus, Sigma-1 receptor activity may contribute to fluvoxamine's potential effects in treating COVID-19, but is not the only mechanism by which this drug can interfere with disease progression.
 
 #### Potential and Limitations of High-Throughput Analyses
 
-These examples illustrate that computational research in drug repurposing can provide some valuable insights, but there are significant limitations to these methods.
-The urgency of the COVID-19 pandemic led many research groups to pivot towards this area of research without familiarity with best practices [@doi:10.1039/D0CS01065K].
+Computational screening allows for a large number of compounds to be evaluated to identify those most likely to display a desired behavior or function.
+This approach can be guided by a hypothesis or can aim to discover underlying characteristics that produce new hypotheses about the relationship between a host, a virus, and candidate pharmaceuticals.
+The examples outlined above illustrate that HTS-based evaluations of drug repurposing can potentially provide valuable insights. 
+Computational techniques were used to design compounds targeting M^Pro^ based on an understanding of how this protease aids viral replication, and M^Pro^ inhibitors remain promising candidates [@doi:10.1039/D0CS01065K], although the clinical trial data is not yet available.
+Similarly, computational analysis correctly identified the Sigma-1 receptor as a protein of interest, and although the process of identifying which drugs might modulate the interaction led to an emphasis on candidates that ultimately have not been supported, fluvoxamine remains an appealing candidate.
+The difference between the preliminary evidence for fluvoxamine compared to other drugs that interact with Sigma receptors underscores a major critique of hypothesis-free HTS in particular: while these approaches allow for brute force comparison of a large number of compounds against a virus of interest, they lose the element of expertise that is associated with most successes in drug repurposing [@doi:10.1021/acs.jcim.0c00861].
+
+There are also practical limitations to these methods.
+One concern is that computational analyses inherently depend on the quality of the data being evaluated.
+The urgency of the COVID-19 pandemic led many research groups to pivot towards computational HTS research without familiarity with best practices in this area [@doi:10.1039/D0CS01065K].
 As a result, there is an excessive amount of information available from computational studies [@url:https://blogs.sciencemag.org/pipeline/archives/2021/07/19/too-many-papers], but not all of it is high-quality.
```

Comment:
This manual reference isn't working. I'll look into it.

Reply:
Thank you Tony!

Reply:
b9ca24f fixed this